app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -30,6 +31,8 @@ data class ValidatedIntentData(val url: String, val source: Session.Source)
 object IntentValidator {
     @VisibleForTesting const val DIAL_PARAMS_KEY = "com.amazon.extra.DIAL_PARAM"
     private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
+    private const val IS_QA_BUILD_KEY = "isQA"
+    private const val FETCH_DELAY_KEY = "fetchDelay"
```

Comment:
nit: I think the value, and the flag QA has to send, should be `qaFetchDelaySeconds`. I think prepending `qa` is important to tell users not to expect much from it and, if we keep `isQA`, to tell QA that that flag is needed to use this flag.